graftm/unpack_sequences.py

```diff
@@ -77,9 +83,17 @@ def sequence_type(self):
                                                 shell=True,
                                                 preexec_fn=lambda:signal(SIGPIPE, SIG_DFL)
                                                 )
-            _, seq = tuple(first_seq.strip().split('\n'))
+            header, seq = tuple(first_seq.strip().split('\n'))
             self.type = self._guess_sequence_type_from_string(seq)
             logging.debug("Detected sequence type as %s" % self.type)
+            
+            slashendingregex = self.slash_ending_regex.match(header)
+            if slashendingregex:
+                logging.debug("Detected '/1' '/2' style headers.")
+                self.slash_endings = True
```

Comment:
I'd prefer boolean variables to be named like `has_slash_endings` or similar. `slash_endings` reads like the variable actually holds the strings `/1` or something.